ksql-engine/src/test/java/io/confluent/ksql/structured/LogicalPlanBuilder.java

```diff
@@ -19,30 +19,27 @@
 import io.confluent.ksql.analyzer.AnalysisContext;
 import io.confluent.ksql.analyzer.Analyzer;
 import io.confluent.ksql.metastore.MetaStore;
-import io.confluent.ksql.parser.KsqlParser;
 import io.confluent.ksql.parser.KsqlParser.PreparedStatement;
+import io.confluent.ksql.parser.KsqlParserTestUtil;
 import io.confluent.ksql.parser.tree.Expression;
 import io.confluent.ksql.parser.tree.ExpressionTreeRewriter;
-import io.confluent.ksql.parser.tree.Statement;
 import io.confluent.ksql.planner.LogicalPlanner;
 import io.confluent.ksql.planner.plan.PlanNode;
 import io.confluent.ksql.util.AggregateExpressionRewriter;
-import java.util.List;
 
 public class LogicalPlanBuilder {
```

Comment:
Want to rename this to something like LogicalPlanBuilderForTest while you're here?

Reply:
Your wish... now called `LogicalPlanBuilderTestUtil`.